### ISSUE TITLE
Question about the preprocessing of obtaining the "ground-truth"

### ISSUE THREAD

**Issue by GWwangshuo** [kierenpinto/ECE4179Project#1]
Hi, I was wondering how do you preprocess the annotation of each expert pathologist to generate the ground-truth label?

Moreover, why you use 4 classes rather than 6 calsses for training purpose? Thanks.

**Comment by kierenpinto** [comment]
1.) Ground truth - I believe they were the average across the expert pathologists. 

2.) Classes - There are only 4 different label types - No cancer (0) Gleason Grade (3),(4),(5). To reduce the complexity of the output softmax, we didn't use 6 classes because 2 wouldn't be used. 

3.) We are new to this, so it is possible that we didn't use the best practices. We followed what we thought was logical and made sense.
